src/jwt/jwt.module.ts

```diff
@@ -0,0 +1,20 @@
+import { Module } from '@nestjs/common';
+import { JwtModule } from '@nestjs/jwt';
+import { JwtAuthService } from './jwt.service';
+import { JwtStrategy } from './jwt.strategy';
+
+@Module({
+  imports: [
+    JwtModule.registerAsync({
+      useFactory: async () => ({
```

Comment:
Should it be an async function?

Reply:
When I tried to use `ConfigModule`, I wrote it to use async.
But, I don't use it. So, will fix it.

Reply:
When I try to use register (not async), JwtModule couldn't get environment variables.
(Error: secretOrPrivateKey must have a value)

So, I think JwtModule need to be an async.

Reply:
The example code in the [nest.js document](https://docs.nestjs.com/fundamentals/custom-providers#non-service-based-providers) uses env variables without async function. Could you check it twice?

Reply:
I thought that it should be async function it for using resgisterAsync.